idex.html

```diff
@@ -11,208 +11,230 @@
+                        fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                    </svg> <span class="project__year">2015</span>
+                </p>
+                <p class="project__paragraph">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</p>
+                <ul class="project__technologies">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button class="project__button">See Project</button>
+            </div>
         </article>
     </section>
     <!-- About me -->
-    <section class="about">
-        <h3 class="about__title">About <br> Myself</h3>
-        <p class="about__paragraph">Hello I’m a software developer! I can help you build a product , feature or website
-            Look through some of my work and experience! If you like what you see and have a project you need coded,
-            don’t hestiate to contact me.</p>
-        <h2 class="about__subtitle">LET'S CONNECT</h2>
-        <ul class="about__icons">
-            <li><img src="./icons/github.svg" alt="github icon"></li>
-            <li><img src="./icons/linkedin.svg" alt="linkedin icon"></li>
-            <li><img src="./icons/angellist.svg" alt="angellist icon"></li>
-            <li><img src="./icons/twitter.svg" alt="twitter icon"></li>
-            <li><img src="./icons/medium.svg" alt="medium icon"></li>
-        </ul>
-        <button class="about__button"><b>Get my resume</b></button>
-        <ul class="knowledge">
-            <li>
-                <div class="knowledge__title">Languages<span class="material-symbols-outlined">
-                        expand_more
-                    </span>
-                </div>
-                <ul class="about__info--technologies">
-                    <li><img class="about__image" src="./images/img-languages-js.jpg" alt="js icon">JavaScript</li>
-                    <li><img class="about__image" src="./images/img-languages-html.jpg" alt="html icon">HTML</li>
-                    <li><img class="about__image" src="./images/img-languages-css.jpg" alt="css icon">CSS</li>
+    <section class="about-content">
+        <div class="about">
+            <div>
+                <h3 class="about__title">About <br> Myself</h3>
+                <p class="about__paragraph">Hello I’m a software developer! I can help you build a product , feature or
+                    website
+                    Look through some of my work and experience! If you like what you see and have a project you need
+                    coded,
+                    don’t hestiate to contact me.</p>
+                <h2 class="about__subtitle">LET'S CONNECT</h2>
+                <ul class="about__icons">
+                    <li><img src="./icons/github.svg" alt="github icon"></li>
```

Comment:
- [ ] kindly add the top right border-radius for ur about section to match the Figma design
![image](https://user-images.githubusercontent.com/89282221/213803347-ff2282c6-851d-4a5e-8ba3-d3ad61eea8a2.png)


Figma:
![image](https://user-images.githubusercontent.com/89282221/213803429-58ec1c47-0110-481a-8c71-cb9132ab0e02.png)